docs/index.md

```diff
@@ -55,10 +55,10 @@ steampipe plugin install gcp
 
 | Item | Description |
 | - | - |
-| Credentials | configure your [Application Default Credentials](https://cloud.google.com/sdk/gcloud/reference/auth/application-default) |
+| Credentials | When running locally, you must configure your [Application Default Credentials](https://cloud.google.com/sdk/gcloud/reference/auth/application-default). If you are running in Cloud Shell, Cloud Code: *"[the tool uses the credentials you provided when you logged in, and manages any authorizations required.](https://cloud.google.com/docs/authentication/provide-credentials-adc#cloud-based-dev)"* |
```

Comment:
```suggestion
| Credentials | When running locally, you must configure your [Application Default Credentials](https://cloud.google.com/sdk/gcloud/reference/auth/application-default). If you are running in Cloud Shell or Cloud Code, [the tool uses the credentials you provided when you logged in, and manages any authorizations required](https://cloud.google.com/docs/authentication/provide-credentials-adc#cloud-based-dev). |
```

My suggestions are mainly style differences, as we don't typically use quotes or italics in cases like these.

Reply:
I used the quotes since I copied the wording straight out of the linked doc. But your suggestion is fine.